force-app/main/default/flows/Create_Event_Registration_Id_Fill_Registration_Fee.flow-meta.xml

```diff
@@ -1,12 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Flow xmlns="http://soap.sforce.com/2006/04/metadata">
     <apiVersion>54.0</apiVersion>
+    <assignments>
+        <name>Cancelled</name>
+        <label>Cancelled</label>
+        <locationX>1076</locationX>
+        <locationY>665</locationY>
+        <assignmentItems>
+            <assignToReference>Session_Registration_Loop.Cancelled__c</assignToReference>
+            <operator>Assign</operator>
+            <value>
+                <booleanValue>false</booleanValue>
+            </value>
+        </assignmentItems>
+        <connector>
+            <targetReference>Create_Unique_Id</targetReference>
+        </connector>
+    </assignments>
+    <assignments>
+        <name>Cancelled_Text</name>
+        <label>Cancelled Text</label>
+        <locationX>757</locationX>
+        <locationY>560</locationY>
+        <assignmentItems>
+            <assignToReference>$Record.Cancelled_Text__c</assignToReference>
+            <operator>Assign</operator>
+            <value>
+                <elementReference>$Record.Cancelled__c</elementReference>
+            </value>
+        </assignmentItems>
+        <connector>
+            <targetReference>Create_Unique_Id</targetReference>
+        </connector>
+    </assignments>
     <assignments>
         <description>Create Unique Id from Public Event Id &amp; Contact Id</description>
```

Comment:
@t-gul Even if only "cancelled" field is changed, you are populating the junction id

Reply:
@sgoEma Yes, I can update this with new decision.

Thanks for review.

Reply:
@sgoEma Flow is updated.